temp_library/helpers.py

```diff
@@ -3,117 +3,103 @@
-    print('welcome to the text adventure: "the warlock of firetop mountain"')
-    input("press enter to start your quest...")
+    opening_screen = ""
+
+    opening_screen += "          /\\\n         /**\\\n        /****\   /\\\n       /      \ /**\\\n      /  /\    /    \ \n     /  /  \  /      \\\n    /  /    \/ /\     \\\n   /  /  _   \/  \/\   \\\n__/__/__( )__/___/__\___\_"
+    opening_screen += '\nwelcome to the text adventure: "the warlock of firetop mountain"'
+
+    return opening_screen
+
 
 
 def statbuild():  # inserts initial values to attributes-change the strings and values according to your specific traits
     """stats setting function
 
     :rtype: Trait
     """
-
     trait1 = temp_library.trait.Trait("Skill", 6, 1)
     trait2 = temp_library.trait.Trait("Luck", 6, 1)
     trait3 = temp_library.trait.Trait("Stamina", 12, 2)
 
     # trait1
-    print("to set your "+trait1.getName()+", please role "+str(trait1.getDiceNum())+" die and add "+str(trait1.getConst()))
+    print("to set your "+trait1.get_name()+", please role "+str(trait1.get_dice_num())+" die and add "+str(trait1.get_const()))
     input("press enter to roll...")
-    print("your " + trait1.getName() + " is in total: " + str(trait1.getValue()))
+    print("your " + trait1.get_name() + " is in total: " + str(trait1.get_value()))
 
     # trait2
-    print("to set your "+trait2.getName()+", please role "+str(trait2.getDiceNum())+" die and add "+str(trait2.getConst()))
+    print("to set your "+trait2.get_name()+", please role "+str(trait2.get_dice_num())+" die and add "+str(trait2.get_const()))
     input("press enter to roll...")
-    print("your " + trait2.getName() + " is in total: " + str(trait2.getValue()))
+    print("your " + trait2.get_name() + " is in total: " + str(trait2.get_value()))
 
     # trait3
-    print("to set your "+trait3.getName()+", please role "+str(trait3.getDiceNum())+" die and add "+str(trait3.getConst()))
+    print("to set your "+trait3.get_name()+", please role "+str(trait3.get_dice_num())+" die and add "+str(trait3.get_const()))
     input("press enter to roll...")
-    print("your " + trait3.getName() + " is in total: " + str(trait3.getValue()))
+    print("your " + trait3.get_name() + " is in total: " + str(trait3.get_value()))
 
     return trait1, trait2, trait3
 
 
 def statsdsply(trait1, trait2, trait3):
     # displaying the stats function
     print(' ------------------------------')
-    print('|'+trait1.getName()+': ' + str(trait1.getValue()) + '|'+trait2.getName()+': '+str(trait2.getValue())+'|'+trait3.getName()+': '+str(trait3.getValue())+'|')
+    print('|'+trait1.get_name()+': ' + str(trait1.get_value()) + '|'+trait2.get_name()+': '+str(trait2.get_value())+'|'+trait3.get_name()+': '+str(trait3.get_value())+'|')
     print(' ------------------------------')
     print()
 
 
-def roll(num, arg):
+def roll(trait, arg):

```

Comment:
Have a better docstring, and example. It isn't really explainable by itself as it.